core/vm/opcodes.go

```diff
@@ -395,10 +388,6 @@ var opCodeToString = map[OpCode]string{
 	REVERT:       "REVERT",
 	INVALID:      "INVALID",
```

Comment:
But for `INVALID` we'll still have `"error": "invalid opcode: INVALID"`, is that the intention here?

Reply:
0xfe is commonly thought as **the** invalid opcode.  So we specially checked `"error": "invalid opcode: INVALID"`, it is ok.

https://www.evm.codes/